isisd/isis_nb_config.c

```diff
@@ -3516,6 +3516,29 @@ int lib_interface_isis_fast_reroute_level_1_ti_lfa_node_protection_modify(
 	return NB_OK;
 }
 
+/*
+ * XPath:
+ * /frr-interface:lib/interface/frr-isisd:isis/fast-reroute/level-1/ti-lfa/link-fallback
+ */
+int lib_interface_isis_fast_reroute_level_1_ti_lfa_link_fallback_modify(
+	struct nb_cb_modify_args *args)
+{
+	struct isis_area *area;
+	struct isis_circuit *circuit;
+
+	if (args->event != NB_EV_APPLY)
+		return NB_OK;
+
+	circuit = nb_running_get_entry(args->dnode, NULL, true);
+	circuit->tilfa_link_fallback[0] =
+		yang_dnode_get_bool(args->dnode, NULL);
+
+	area = circuit->area;
+	lsp_regenerate_schedule(area, area->is_type, 0);
```

Comment:
area may not exist yet.

Should be like in all other callbacks:
```
if (area)
        lsp_regenerate_schedule(area, area->is_type, 0);
```

Reply:
@idryzhov Thanks for pointing this out! I cherry-picked the commit from an out-dated branch that did not have those sanities and did not realize about that. Will amend the code ASAP.